boomi/boomi-workflow-parameter-values.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<WorkflowParameterValues xmlns="http://qradar.ibm.com/UniversalCloudRESTAPI/WorkflowParameterValues/V2">
+	<!-- [identifier] [REQUIRED] - The log source identifier to post the events to. -->
+	<Value name="identifier" value="Boomi"/>
+	<!-- [account_id] [REQUIRED] - Boomi account ID.-->
+	<Value name="account_id" value=""/>
+	<!-- [username] [REQUIRED] - Boomi user name. -->
+	<Value name="username" value=""/>
+	<!-- [token] [REQUIRED] - Boomi token. -->
+	<Value name="token" value=""/>
+	<!-- [time_zone] [OPTIONAL] [DEFAULT=UTC] - The time zone to be used. -->
+	<Value name="time_zone" value="UTC"/>
+	<!-- [start_fetch_time] [REQUIRED] - The date time from which events will be initially retrieved. -->
+	<Value name="start_fetch_time" value="2021-10-01T00:00:00Z"/>

```

Comment:
we need to leave it empty and config default time for the first run.

Reply:
This is a required parameter so why configure a default start time?
We can change the parameter to be optional.